### PR TITLE
Manager connect flow

### PR DESCRIPTION
**Pull request by juan-cortes**
This doesn't have the UI yet but should allow access to the manager with the new connect flow without the device interaction stepper. I'm pending the UI until we figure out if we will use animations or not, and if we'll use generic or specific device illustrations.

### Type

Feature

### CODE REVIEW

File: src/internal/commands/connectManager.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        if (deviceInfo.isBootloader || deviceInfo.isOSU) {
+          return of({ deviceInfo, event: {} }); // Maybe generic "not in dashboard state" event
+        }
+
+        // Preload things in parallel
+        manager.getLatestFirmwareForDevice(deviceInfo).catch(e => {
+          logger.warn(e);
+        });
+
+        return listApps(transport, deviceInfo).pipe(map(event => ({ event, deviceInfo })));
```
Comment (gre) [inline src/internal/commands/connectManager.js]: `map(..` can be replaced by the `scan(..` code that happened below without the need to make a nested object

File: src/renderer/i18n/en/app.json [hunk 2]
```diff
@@ -429,8 +429,10 @@
       },
       "sort": {
         "title": "Sort",
-        "name": "Name",
-        "marketcap": "Marketcap"
+        "name_asc": "Name ASC",
```
Comment (gre) [inline src/renderer/i18n/en/app.json]: Name A-Z
Name Z-A

File: src/renderer/i18n/en/app.json [hunk 3]
```diff
@@ -429,8 +429,10 @@
...
+        "name_desc": "Name DESC",
+        "marketcap_asc": "Marketcap ASC",
+        "marketcap_desc": "Marketcap DESC"
```
Comment (gre) [inline src/renderer/i18n/en/app.json:435]: do we need the one from "smaller to highest"?

**Review by gre — APPROVED** [review]

File: src/renderer/i18n/en/app.json (hunk 3 above)
Comment (juan-cortes) [reply src/renderer/i18n/en/app.json:435]: @bscholving your call as far as I'm concerned

File: src/internal/commands/connectManager.js (hunk 1 above)
Comment (juan-cortes) [reply src/internal/commands/connectManager.js]: Scan used to be outside of the concatMap, what you're saying now makes sense indeed.

**Comment by juan-cortes** [comment]
🤧 @gre I've made the requested changes, hopefully, we can merge this so I can make another pr with UI without things getting messy

**Review by gre — APPROVED** [review]

**Comment by gre** [comment]
@juan-cortes conflicts

**Review by gre — APPROVED** [review]

**Pull request merged**